shell/common/shell.cc

```diff
@@ -1967,7 +1975,9 @@ bool Shell::OnServiceProtocolRenderFrameWithRasterStats(
 
     response->AddMember("snapshots", snapshots, allocator);
 
-    const auto& frame_size = expected_frame_size_;
+    // TODO(dkwingsmt): Not sure what these fields are supposed to be after
+    // multi-view. For now it sends the implicit view's dimension as before.
```

Comment:
Should we create a tracking issue for this? We will likely forget to fix this otherwise until something breaks

Reply:
Done. Issue at https://github.com/flutter/flutter/issues/131892.